src/packet/header/standard_header.rs

```diff
@@ -63,6 +63,11 @@ impl StandardHeader {
         self.packet_type == PacketType::Fragment
     }
 
+    /// Returns true if packet is only used by `ConnectionManager`
+    pub fn is_connection_manager_packet(&self) -> bool {
```

Comment:
same here, below we had manager packet, and here we have connection manager packet.

Reply:
renamed to `is_connection_packet`